app/actions/core.py

```diff
@@ -11,6 +11,9 @@ class PullActionConfiguration(ActionConfiguration):
     pass
 
 
+class ExecutableActionMixin:
+    pass
```

Comment:
Thank you for adding this. I'll add a reference to this in the [story we have for improvements in the integrations template](https://allenai.atlassian.net/browse/GUNDI-3496).